File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/MainEstimateDealBody.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class MainEstimateDealBody {
    /// <summary>
    /// Gets or Sets DurationBlks
    /// </summary>
    [DataMember(Name="durationBlks", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "durationBlks")]
    public int? DurationBlks { get; set; }

    /// <summary>
    /// Gets or Sets Replication
    /// </summary>
    [DataMember(Name="replication", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "replication")]
    public int? Replication { get; set; }

    /// <summary>
    /// Gets or Sets Size
    /// </summary>
    [DataMember(Name="size", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "size")]
    public int? Size { get; set; }

    /// <summary>
    /// Gets or Sets Verified
    /// </summary>
    [DataMember(Name="verified", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "verified")]
    public bool? Verified { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MainEstimateDealBody {\n");
      sb.Append("  DurationBlks: ").Append(DurationBlks).Append("\n");
      sb.Append("  Replication: ").Append(Replication).Append("\n");
      sb.Append("  Size: ").Append(Size).Append("\n");
      sb.Append("  Verified: ").Append(Verified).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/main_estimate_deal_body.dart
part of swagger.api;

class MainEstimateDealBody {
  
  int durationBlks = null;

  int replication = null;

  int size = null;

  bool verified = null;

  MainEstimateDealBody();

  @override
  String toString() {
    return 'MainEstimateDealBody[durationBlks=$durationBlks, replication=$replication, size=$size, verified=$verified, ]';
  }

  MainEstimateDealBody.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    durationBlks = json['durationBlks'];
    replication = json['replication'];
    size = json['size'];
    verified = json['verified'];
  }

  Map<String, dynamic> toJson() {
    return {
      'durationBlks': durationBlks,
      'replication': replication,
      'size': size,
      'verified': verified
     };
  }

  static List<MainEstimateDealBody> listFromJson(List<dynamic> json) {
    return json == null ? new List<MainEstimateDealBody>() : json.map((value) => new MainEstimateDealBody.fromJson(value)).toList();
  }

  static Map<String, MainEstimateDealBody> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MainEstimateDealBody>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MainEstimateDealBody.fromJson(value));
    }
    return map;
  }
}
